proto/spire/api/agent/delegatedidentity/v1/delegatedidentity.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package spire.api.agent.delegatedidentity.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/agent/delegatedidentity/v1;delegatedidentityv1";
+
+import "spire/api/types/selector.proto";
+import "spire/api/types/x509svid.proto";
+
+// The delegatedIdentity service provides an interface to get the SVIDs of other
+// workloads on the host. This service is intended for use cases where a process
+// (different than the workload one) should access the workload's SVID to
+// perform actions on behalf of the workload. One example of is using a single
+// node instance of Envoy that upgrades TCP connections for different processes
+// running in such a node.
+//
+// The caller must be local and its identity must be listed in the allowed
+// clients on the spire-agent configuration.
+service DelegatedIdentity {
+    // Subscribe to get X.509-SVIDs for identities that match the given selectors.
```

Comment:
Since we are delegating workload SVIDs here, I think it is natural to frame the comment as obtaining a workload's SVIDs rather than obtaining an identity's SVIDs

```suggestion
    // Subscribe to get X.509-SVIDs for workloads that match the given selectors.
```